book_source/installation/thredds.Rmd

```diff
@@ -0,0 +1,341 @@
+# Installing and configuring Thredds for PEcAn
+authors - Rob Kooper
+
+**NOTE: Instructions are only tested for Ubuntu 16.04 on the VM.**
+**NOTE: Pretty much every step here requires root access.**
+
+## Install the Tomcat 8 and Thredds webapp
+
+The Tomcat 8 server can be installed from the default Ubuntu repositories. The thredds webapp will be downloaded and installed from unidata.
+
+## Ubuntu
+
+First step is to install Tomcat 8 and configure it.
+
+```
+apt-get -y install tomcat8 openjdk-8-jdk
+echo JAVA_OPTS=\"-Dtds.content.root.path=/home/carya \${JAVA_OPTS}\" >> /etc/default/tomcat8
+echo JAVA_OPTS=\"-Djava.security.egd=file:/dev/./urandom \${JAVA_OPTS}\" >> /etc/default/tomcat8
+service tomcat8 restart
+```
```

Comment:
This should be to install tomcat on the machine and configure it to find the thredds catalog. For speed purposes on the VM the random number generator for tomcat is set to urandom, and not random.